File: backjoon/완전탐색/JS/한수.js
const input = require("fs")
  .readFileSync(process.platform === "linux" ? "./dev/stdin" : "./input.txt")
  .toString()
  .trim()
  .split("\n");

const n = +input.shift();

let hansuCount = 0;

for (let i = 1; i <= n; i++) {
  if (i < 100) {
    hansuCount++;
  } else {
    let array = String(i).split("").map(Number);
    if (array[0] - array[1] === array[1] - array[2]) {
      hansuCount++;
    }
  }
}

console.log(hansuCount);

File: backjoon/완전탐색/Python/한수.py
def solve() :

    hansuCount = 0

    for i in range(1, N+1) :
        ## 숫자를 문자열로 분리한 후에 다시 숫자로 변환
        num_list = list(map(int, str(i)))
        if(i < 100) :
            hansuCount += 1
        elif num_list[0] - num_list[1] == num_list[1] - num_list[2] :
            hansuCount += 1    

    print(hansuCount)

if __name__ == "__main__":
    N = int(input()) #110
    solve()